terraform/staging/main.tf

```diff
@@ -40,6 +40,7 @@ locals {
   app_host         = "staging.submit-social-housing-data.levellingup.gov.uk"
   application_port = 8080
   database_port    = 5432
+  load_balancer_domain_name = "staginglb.submit-social-housing-data.levellingup.gov.uk"

```

Comment:
Can I suggest this is `staging.lb.submit-social-housing-data.levellingup.gov.uk`? (It doesn't really make a difference, just is a nicer pattern of making it always `staging.whatever-prod-is`)